variant-remapping-automation/remapping_automation.py

```diff
@@ -0,0 +1,200 @@
+        return mongo_host, mongo_user, mongo_pass
+
+    @staticmethod
+    def get_accession_pg_creds():
+        properties = get_properties_from_xml_file(cfg['maven']['environment'], cfg['maven']['settings_file'])
+        pg_url = properties['eva.accession.jdbc.url']
+        pg_user = properties['eva.accession.user']
+        pg_pass = properties['eva.accession.password']
+        return pg_url, pg_user, pg_pass
+
+    @staticmethod
+    def write_remapping_process_props_template(template_file_path):
+        mongo_host, mongo_user, mongo_pass = RemappingJob.get_mongo_creds()
+        pg_url, pg_user, pg_pass = RemappingJob.get_accession_pg_creds()
+        with open(template_file_path, 'w') as open_file:
+            open_file.write(f'''spring.datasource.driver-class-name=org.postgresql.Driver
+spring.datasource.url={pg_url}
+spring.datasource.username={pg_user}
+spring.datasource.password={pg_pass}
+spring.datasource.tomcat.max-active=3
+
+spring.jpa.generate-ddl=true
+
+spring.data.mongodb.host={mongo_host}
+spring.data.mongodb.port=27017
+spring.data.mongodb.database=eva_accession_sharded
+spring.data.mongodb.username={mongo_user}
+spring.data.mongodb.password={mongo_pass}
+
+spring.data.mongodb.authentication-database=admin
+mongodb.read-preference=secondaryPreferred
+spring.main.web-environment=false
+spring.main.allow-bean-definition-overriding=true
+spring.jpa.properties.hibernate.jdbc.lob.non_contextual_creation=true
+logging.level.uk.ac.ebi.eva.accession.remapping=INFO
+''')
+        return template_file_path
+
+    @staticmethod
+    def get_metadata_conn():
+        pg_url, pg_user, pg_pass = RemappingJob.get_metadata_creds()
+        return psycopg2.connect(urlsplit(pg_url).path, user=pg_user, password=pg_pass)
+
+    def get_job_information(self, assembly):
+        query = (
+            'SELECT source, scientific_name, taxid, target_assembly_accession, SUM(number_of_study), '
+            'SUM(number_submitted_variants) '
+            'FROM remapping_progress '
+            f"WHERE assembly_accession='{assembly}' "
+            'GROUP BY source, assembly_accession, scientific_name, taxid, target_assembly_accession'
+        )
+        source_list = []
+        scientific_name = None
+        taxid = None
+        target_assembly = None
+        n_study = 0
+        n_variants = 0
+        with self.get_metadata_conn() as pg_conn:
+            for source, scientific_name, taxid, target_assembly, n_st, n_var in get_all_results_for_query(pg_conn, query):
+
+                source_list.append(source)
+                n_study += n_st
+                n_variants += n_var
+        sources = ', '.join(source_list)
+        self.info(f'Process assembly {assembly} for taxonomy {taxid}: {n_study} studies with {n_variants} '
+                   f'found in {sources}')
+        return sources, scientific_name, taxid, target_assembly, n_study, n_variants
+
+    def list_assemblies_to_process(self):
```

Comment:
I guess you're still testing this which is why this isn't being used, right?

Reply:
Yes that's right. This was more of a place holder.
It is now being used but there is still some work required to figure out what we want to show